### PR TITLE
DOC-6248: Corda 4.12 version update

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Four pages to be updated:

1. **Corda Enterprise 4.12 platform support:** https://niamh.preview.docs.r3.com/en/platform/corda/4.12/enterprise/platform-support-matrix.html
2. **Host prerequisites and database requirements:** https://niamh.preview.docs.r3.com/en/platform/corda/4.12/enterprise/node/setup/host-prereq.html
3. **code snippets in Publishing versions in your JAR manifest files:** 
- ENT: https://niamh.preview.docs.r3.com/en/platform/corda/4.12/enterprise/cordapps/versioning.html#publishing-versions-in-your-jar-manifest-files
- COMM: https://niamh.preview.docs.r3.com/en/platform/corda/4.12/community/versioning.html#publishing-versions-in-your-jar-manifests

### CODE REVIEW

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md [hunk 1]
```diff
@@ -8,24 +8,24 @@ tags:
...
 ### JDK support in production
 
-Corda Enterprise Edition 4.11 has been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**, for Azure deployment downloadable from
+Corda Enterprise Edition 4.12 has been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**, for Azure deployment downloadable from
 [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
 
-Other distributions of the [OpenJDK](https://openjdk.java.net/) are not officially supported but should be compatible with Corda Enterprise Edition 4.11.
+Other distributions of the [OpenJDK](https://openjdk.java.net/) are not officially supported but should be compatible with Corda Enterprise Edition 4.12.
 
 {{< warning >}}
```
Comment (tomstark99) [inline content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:30]: Remove this warning block

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md
Comment (tomstark99) [inline content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:1]: Line 38, does not support java 18 or higher

**Review by tomstark99 — CHANGES_REQUESTED** [review]
I am still waiting on feedback on which HSM driver versions were used in testing but these changes can be done now already

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md [hunk 2]
```diff
@@ -8,24 +8,24 @@ tags:
...
+This topic lists the JDKs, operating systems and database types that support Corda Enterprise Edition 4.12, both for production and for development. For the 'end of life schedule' for Corda versions and the associated documentation, refer to the [End of Life Schedule]({{< relref "../../eol-schedule.md" >}}).
 
 ## JDK support
 
 Corda Enterprise Edition supports a subset of the platforms that are supported by [Java](http://www.oracle.com/technetwork/java/javase/certconfig-2095354.html).
 
 ### JDK support in production
 
-Corda Enterprise Edition 4.11 has been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**, for Azure deployment downloadable from
+Corda Enterprise Edition 4.12 has been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**, for Azure deployment downloadable from
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:25]: Updated

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:30]: Removed

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:1]: Removed and updated

Comment (AnnaK-R3) [reply content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:1]: Updated

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md [hunk 3]
```diff
@@ -35,24 +35,21 @@ for your deployment needs.
...
 
 {{< table >}}
 
 |Supported JDKs|Latest supported version|
 |-----------------------------------|-----------|
-|[Oracle JDK](https://www.oracle.com/ie/java/technologies/downloads/)|8u381|
-|[Amazon Corretto 8](https://aws.amazon.com/corretto/)|8.252.09.1|
-|[Red Hat OpenJDK](https://developers.redhat.com/products/openjdk/overview/)|8u322|
-|[Zulu OpenJDK](https://www.azul.com/)|8u382|
+|[Zulu OpenJDK](https://www.azul.com/)|17.0.9|
```
Comment (tomstark99) [inline content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:44]: _Sorry_ Anna, 

|[Oracle JDK](https://www.oracle.com/ie/java/technologies/downloads/)|17.0.9|

needs to be added back in now

File: content/en/platform/corda/4.12/enterprise/platform-support-matrix.md [hunk 4]
```diff
@@ -8,24 +8,24 @@ tags:
...
 ### JDK support in production
 
-Corda Enterprise Edition 4.11 has been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**, for Azure deployment downloadable from
+Corda Enterprise Edition 4.12 has been tested and verified to work with **Azul Zulu Enterprise 17.0.9**, for Azure deployment downloadable from
 [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
 
-Other distributions of the [OpenJDK](https://openjdk.java.net/) are not officially supported but should be compatible with Corda Enterprise Edition 4.11.
+Other distributions of the [OpenJDK](https://openjdk.java.net/) are not officially supported but should be compatible with Corda Enterprise Edition 4.12.
 
 {{< warning >}}
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/4.12/enterprise/platform-support-matrix.md:30]: Removed

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**